src/bulletin/bulletin.service.ts

```diff
@@ -91,4 +91,54 @@ export class BulletinService {
       }
     });
   }
+
+  async deleteBulletin(id: number) {
+    try {
+      const dateNow = new Date();
+      const bulletin = await this.prismaService.bulletin.findUnique({
+        where: { id },
+      });
+
+      if (!bulletin) {
```

Comment:
Validation of the bulletin ID should be handled in the DTO. You can see an example made by AJ in the `createBulletin.dto.ts`